### PR TITLE
CAM-14285: document disabled remote access to h2 console

### PR DESCRIPTION
**Pull request by ThorbenLindhauer**
related to CAM-14285

### CODE REVIEW

File: content/update/minor/716-to-717/_index.md [hunk 1]
```diff
@@ -122,4 +123,26 @@ LDAP user group gains administrative access to configure additional authorizatio
 
 The Administrator Authorization plugin configuration properties are integrated with Camunda Run. You can find them
 in the [LDAP Adminstrator Authorization section]({{< ref "/user-guide/camunda-bpm-run.md#ldap-administrator-authorization" >}})
-of the Camunda Run documentation. You can also find a template LDAP configuration in the Camunda Run `production.yml`.
\ No newline at end of file
+of the Camunda Run documentation. You can also find a template LDAP configuration in the Camunda Run `production.yml`.
+
+# Disabled remote access to H2 console
+
+This set of patches deactivates remote access to the H2 console application in the Tomcat and Wildfly distributions. The H2 application accepts only localhost connections moving forward.
```
Comment (tmetzke) [inline content/update/minor/716-to-717/_index.md:130]:
```suggestion
The Camunda Platform 7.17.0 release deactivates remote access to the H2 console application in the Tomcat and Wildfly distributions. The H2 application accepts only localhost connections moving forward.
```

File: content/update/minor/716-to-717/_index.md [hunk 2]
```diff
@@ -122,4 +123,26 @@ LDAP user group gains administrative access to configure additional authorizatio
...
+To restore remote access, add the following initialization parameter to the `org.h2.server.web.WebServlet` servlet defined in the `web.xml` file of the h2 web application:
+
+```
+<init-param>
+  <param-name>webAllowOthers</param-name>
+  <param-value>true</param-value>
+</init-param>
+```
+
+`web.xml` is located in the following paths:
```
Comment (tmetzke) [inline content/update/minor/716-to-717/_index.md:141]:
```suggestion
You can find the `web.xml` at the following paths:
```

File: content/update/patch-level.md [hunk 3]
```diff
@@ -309,6 +309,28 @@ You can restore the old behavior by passing the appropriate [configuration prope
...
+To restore remote access, add the following initialization parameter to the `org.h2.server.web.WebServlet` servlet defined in the `web.xml` file of the h2 web application:
+
+```
+<init-param>
+  <param-name>webAllowOthers</param-name>
+  <param-value>true</param-value>
+</init-param>
+```
+
+`web.xml` is located in the following paths:
```
Comment (tmetzke) [inline content/update/patch-level.md:325]:
```suggestion
You can find the `web.xml` at the following paths:
```

**Review by tmetzke — CHANGES_REQUESTED** [review]
I like the pragmatic and concise description 👍 
I only added some grammatical suggestions.

**Review by tmetzke — APPROVED** [review]

**Review by christinaausley — APPROVED** [review]
This looks great -- excellent work and no edits on my end.

Note that you can remove "please" from "please note" moving forward to keep things concise. You can just say "Note that we strongly discourage..."

By the way, I know we have :::note admonitions in Cloud, but do we have anything of the likes within Platform? Just curious.

**Pull request merged**